### DISCUSSION TITLE
Confusion over setting "Show sizes"

### DISCUSSION THREAD

**Discussion by mdoss** [TeamAmaze/AmazeFileManager#4126] [General]
The setting description says it shows how many files are in each folder but I don't see that anywhere. Am I misinterpreting what this setting is supposed to do? I would prefer if I could see how many files are in each folder.

**Comment by VishnuSanal** [comment]
this setting item enables/disables the "file sizes" on file listing. and, the file count for directory seems to be missing (or broken) somehow, we'll look into it, thanks!

@VishalNehra @EmmanuelMess was this always the case? did we use to show how many files were inside a directory before?
